apostrophecms/apostrophe
# Pro 2744 date and time schema field

[PRO-2744](https://linear.app/apostrophecms/issue/PRO-2744/create-the-dateandtime-schema-field-type)

## Summary
Creates a new `dateAndTime` schema field

## What are the specific steps to test this change?

* Link this banch in an A3 project (testbed)
* Create a field in a module of type `dateAndTime`
* Manipulate this field, it should:
   - Save a date of at least a date is set (if no time it will be 00:00:00)
   - when adding a date, the toggle should automatically pass on on mode.
   - of toggle off no date is saved

## What kind of change does this PR introduce?


## Make sure the PR fulfills these requirements:
